### ISSUE TITLE
Handle single-quoted attributes properly

### ISSUE THREAD

**Issue by mDuo13** [tvler/streetpass#15]
It seems the [Firefox] extension doesn't find the Mastodon link when you specify it with the HTML attributes using single quotes, like so:

```html
<link rel='me' href='https://example.com/@username' />
```

It worked fine immediately after switching it over to double-quotes, like so:

```html
<link rel="me" href="https://example.com/@username" />
```

Interestingly, [your source code](https://github.com/tvler/streetpass/blob/392c7af08583197e9e970ce1d757c91b65426e99/extension/src/content-script.ts#L15) doesn't seem to be checking the quotes at all:

```
document.querySelectorAll(":is(link, a)[rel~=me]");
```

And this line runs fine in the developer console regardless of what type of quotes are used in the page. So, my only conclusion is that there must be a bug somewhere in the webpack pipeline or something that's causing this line to behave incorrectly—perhaps calling a polyfill/imitation of `querySelectorAll` instead of the real thing—in the extension.

**Comment by tvler** [comment]
was this item already in your streetpass list when you weren't seeing it pop up with single quotes? Asking because if something is already in your list, it won't show the notification again

Interesting tho. The build step doesn't add any pollyfills besides one that unifies all the cross browser extension APIs so it's not that. I could check out in my free time

**Comment by mDuo13** [comment]
> was this item already in your streetpass list when you weren't seeing it pop up with single quotes? Asking because if something is already in your list, it won't show the notification again

No, it was a brand new entry (my own personal site). Until I changed the code, the only entry on my streetpass list was the sample/test one (tvler)
